src/components/templates/PartyMap/components/PartyMapPreview/PartyMapPreview.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FC } from "react";
+import { PartyMapRoomData } from "types/PartyMapRoomData";
+
+interface PartyMapPreviewProps {
+  expanded: boolean;
+  backgroundImage?: string;
+  backgroundRatio: number;
+  avatarPosition: {
+    x: number;
+    y: number;
+  };
```

Comment:
This can use `Point` from `src/types/utility.ts`
```suggestion
  avatarPosition: Point;
```